src/_layouts/base.html

```diff
@@ -194,6 +194,11 @@
 <script src="{{ url_for('static', filename='js/routes/common.min.js') }}"></script>
 {% endblock javascript %}
 
+<script type='text/javascript' id="__bs_script__">
```

Comment:
Ah, thanks for the reminder. I have a semi-working solution to keeping this local and I'll get it finished now.